ChameleonForms/Attributes/RequiredFlagsEnumAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace ChameleonForms.Attributes
+{
+    /// <summary>
+    /// Marks a Flags enum property as required.
+    /// </summary>
+    public class RequiredFlagsEnumAttribute : RequiredAttribute
+    {
+        /// <inheritdoc />
+        public override bool IsValid(object value)
+        {
+            if (Convert.ToInt64(value) == 0L)

```

Comment:
Flags enums have a default value of 0, which didn't trigger as not valid for a required field with the default attribute.